packages/@orchidui-vue/src/DataDisplay/Table/OcTable.vue

```diff
@@ -199,100 +215,120 @@ onMounted(() => onScroll());
       </div>
     </template>
     <template v-else>
-      <div
-        v-for="(field, i) in fields"
-        :key="i"
-        :class="{
-          'border-b': fields.length !== i + 1,
-        }"
+      <Draggable
+        v-slot="{ list }"
+        :model-value="fields"
+        class="grid gap-3"
+        @update:model-value="$emit('update:modelValue', $event)"
```

Comment:
here we need to wrap the content inside slot with name 
```suggestion
      <slot
        name="table-body"
        :fields="fields"
        :is-scroll-on-end="isScrollOnEnd"
        :on-click-row="onClickRow"
        :another-props="anotherPropsCanPassIntoParentComponents"
      >
        <div
          v-for="(field, i) in fields"
          :key="i"
          :class="{
            'border-b': fields.length !== i + 1
```

Reply:
In parent component, which we need to use with slot
```
<oc-table
  :options="options"
  :is-sticky="true"
>
  <template #table-body="{ fields, onClickRow , isScrollOnEnd, anotherProps, ...etc }">

    // Here we do custom slot
    <Draggable
        v-slot="{ list }"
        :model-value="fields"
        class="grid gap-3"
        @update:model-value="doSomething"
    >
      <div
        v-for="(field, i) in fields"
        :key="i"
      >
        <TableCell
          v-for="(header, j) in headers"
          :key="`${j}-${i}`"
        >
          // cell data here
        </TableCell>
      </div>
    </Draggable>
  </template>
</oc-table
```

Reply:
@HoaiLee @kewcoder added table-header and table-body slots. Please, take a look.